web/themes/custom/genlyd_aarhus/source/_patterns/01-modules/progress-bar.twig

```diff
@@ -1,10 +1,23 @@
 <div class="progress-bar">
```

Comment:
So what is this: https://github.com/aakb/genlydaarhus/blob/develop/web/themes/custom/genlyd_aarhus/templates/misc/status-messages.html.twig